### ISSUE TITLE
Warning: two versions found on macOS

### ISSUE THREAD

**Issue by henryiii** [jwlawson/actions-setup-cmake#26]
I've noticed a warning showing up:

```
Run jwlawson/actions-setup-cmake@v1.7
  with:
    github-api-token: ***
  env:
    pythonLocation: /Users/runner/hostedtoolcache/Python/3.5.10/x64
    BOOST_ROOT: 
Warning: Found 2 matching packages.
/usr/bin/tar xz -C /Users/runner/work/_temp/6b8603ae-eb43-4975-bbd4-ea6affe3299c -f /Users/runner/work/_temp/34ac4217-2c62-4652-9e92-b47f7bf78452
```

See, for example, https://github.com/pybind/pybind11/runs/1773428287?check_suite_focus=true

Is this due to Universal + regular, perhaps?

**Comment by jwlawson** [comment]
I guess that this is down to the two Mac packages:
* cmake-3.19.4-macos-universal.dmg
* cmake-3.19.4-macos10.10-universal.dmg

https://github.com/Kitware/CMake/releases/tag/v3.19.4

Looks like this comes from https://github.com/Kitware/CMake/commit/6410425e5b036dc643ea40343899c04cf8b2fc85 so both packages should work with the MacOS versions available on the runners (10.5 and 11.0).

**Comment by Vollstrecker** [comment]
Just stumbled across this also. I've choosen 3.20.1 t avoid multiple match with the rc's of 3.20.0, but it still complains. Is there any way of getting rid f this? Would be great as this are the only warnings left so far.

**Comment by jwlawson** [comment]
Yeah I guess a warning that happens all the time isn't super useful. #56 should resolve this, available in release v1.14.0
